src/services/contentfulClient.js

```diff
@@ -10,22 +11,29 @@ function initClient (space, accessToken, preview) {
     host: preview ? 'preview.contentful.com' : 'cdn.contentful.com'
   })
   return client.getSpace()
-  .then(() => {
-    authorized = true
-    return authorized
-  })
+    .then((space) => {
+      authorized = true
+      currentSpace = space
+      return space
+    })
 }
 
 function getClient () {
   return authorized && client
 }
 
+function getCurrentSpaceName () {
+  let currentSpaceName = (currentSpace && currentSpace.name) ? currentSpace.name : ''
+  if (currentSpaceName.length > 13) {
```

Comment:
I saw this `...` logic several times... maybe we wanna create a util or something?